### ISSUE TITLE
Unibeautify 0.9.0 is removing newlines

### ISSUE THREAD

**Issue by szeck87** [Unibeautify/unibeautify#42]
See https://travis-ci.org/Unibeautify/beautifier-eslint/jobs/346110253.  Notice how the received output is missing a newline.

Expected value to be:
```
"console.log('hello world');
console.log('hello world');
"
```

Received:
```
 "console.log('hello world');
console.log('hello world');"
```

**Comment by Glavin001** [comment]
THanks! Working on it now.

**Comment by Glavin001** [comment]
Published to v0.9.1
